MAINT: Common library migration (#254)
ansys/pyaedt-toolkits-antenna: doc/source/getting_started/user_guide.rst

==========
User guide
==========

> You have multiple options for installing and launching this toolkit:

Reviewer: ```suggestion
You have multiple options for installing and launching the AEDT Antenna Toolkit:
```

> - You can install the toolkit directly in AEDT via an installation script and then launch it as a wizard.

Reviewer: ```suggestion
- You can install the toolkit directly in AEDT using an installation script and then launch it as a wizard.
```
Our style guide says not to use via.

  For more information, see :ref:`install-toolkit-AEDT`.

> - You can install the toolkit from a Python console and then launch Antenna wizard.

Reviewer: ```suggestion
- You can install the toolkit from a Python console and then launch the Antenna wizard.
```
If wizard is capitalized in the UI's title bar, then it should always be capitalized as a product name here.

  For more information, see :ref:`install_toolkit_console_ui`.
- You can install the toolkit from a Python console and then use the toolkit's APIs.
  For more information, see :ref:`install_toolkit_console_api`.

.. _install-toolkit-AEDT:

Install toolkit in AEDT and launch the Antenna wizard
-----------------------------------------------------

> You can install the Antenna Toolkit directly in AEDT using the base

Reviewer: ```suggestion
You can install the AEDT Antenna Toolkit directly in AEDT using the base
```
We have some naming issues that should be cleared up in the doc in a separate PR. Based on the repo name and the title on the doc landing page, the name of this product is AEDT Antenna Toolkit. It should always be written and capitalized like that. Once this PR is merged, ping me to do an overall documentation review.

interpreter from the AEDT installation.

> #. From `Install from a Python file <https://aedt.docs.pyansys.com/version/stable/Getting_started/Installation.html#install-from-a-python-file>`_,

Reviewer: You should always tell a user where the link is going to take them so that they can decide if they want to click it. This is why the "in the PyAEDT documentation was there. I'd suggest keeping it there.

   follow the steps to install PyAEDT inside AEDT.

> #. In AEDT, select Tools > Toolkit > PyAEDT > Console to load the PyAEDT console:

Reviewer: ```suggestion
#. In AEDT, select **Tools > Toolkit > PyAEDT > Console** to load the PyAEDT console:
```
UI elements should always be in bold.

       desktop.add_custom_toolkit("AntennaWizard")
       exit()

#. In the AEDT toolbar, click the **AntennaWizard** button to open this wizard in AEDT:

   .. image:: ../_static/antenna_catalog.png
      :width: 800
      :alt: Antenna Toolkit in AEDT

> The Antenna Toolkit Wizard is connected directly to the AEDT session. For wizard usage information,

Reviewer: This is another naming inconsistency. Earlier, there was a reference to Antenna Wizard. We need to match the "product name" always.

see :doc:`../toolkit/ui`.

.. _install_toolkit_console_ui:

> Install toolkit from Python console and launch the Antenna wizard

Reviewer: Proper wizard name? Wizard should probably be capitalized. Is Toolkit in it or not?

-----------------------------------------------------------------

> You can install the Antenna Toolkit in a specific Python environment from the AEDT console.

Reviewer: ```suggestion
You can install the AEDT Antenna Toolkit in a specific Python environment from the AEDT console.
```

> #. On the **AEDT Settings** tab, create a new AEDT session or connect to an existing one:

Reviewer: ```suggestion
#. On the **AEDT Settings** tab, create an AEDT session or connect to an existing one:
```
